### ISSUE TITLE
All mutants survive with TypeScript 3.9 on Windows

### ISSUE THREAD

**Issue by nicojs** [stryker-mutator/stryker#2214]
**Summary**

All mutants always survive on Windows with typescript transpiler enabled and TypeScript 3.9. 

For example, checkout Stryker itself and do the following commands:

```
npm i
npm run build
cd packages/mocha-runner
npm run stryker
```

**Stryker config**

```json
{
   "transpilers": ["typescript"]
}
```

**Test runner config**

**Stryker environment**

```
typescript: 3.9.3
```

**Your Environment**

| software         | version(s)
| ---------------- | -------
| node             |
| npm              |
| Operating System | windows

**Comment by nicojs** [comment]
@simondel about time we configure a windows build server. At least for the e2e tests. Should be strait forward now with GH actions

**Comment by nicojs** [comment]
Closed with #2215. Fixed in `@stryker-mutator/typescript@3.2.4`
